templates/options.erb

```diff
@@ -0,0 +1,15 @@
+Dpkg::Options {
+<%- @_options.each_pair do |config, value| 
+  if %w(force_confdef force_confold force_confnew force_confmiss).include?(config) then
+    if value.is_a?(TrueClass) || value.is_a?(FalseClass) then 
+      if value then -%>
```

Comment:
> We check if it is a boolean or not so that we can provide a usefull error message to the user i.e. this value must be a boolean. We then we check if the value is true.

Ah, I see what you're getting at.

And yes, I'd be happier with 4 `validate_bool` calls actually, catch it as early as possible so we keep the logic in the template to a minimum. This could further be improved once we start moving to Puppet 4 only for our modules and leverage the type system instead.